### ISSUE TITLE
Many PVs come up in invalid state

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/lcls-plc-lfe-motion#18]
`caget IM1L1:PPM:SPM:VOLT_RBV.STAT` is always good
`caget IM3L0:PPM:SPM:VOLT_RBV.STAT` is always UDF

In the PLC these are both working and have a value. They are pragma'd the same way as well.

Overall, 1715 PVs always come up UDF and 3893 always come up ok. More investigation needed.

**Comment by ZLLentz** [comment]
TODO: separate UDF and ok PVs by whether they are input or output records. Perhaps we get a clean number of "working" pvs if we only look at input records?

**Comment by ZLLentz** [comment]
Separating by record type was not useful, needs closer debugging look

**Comment by ZLLentz** [comment]
After talking to Mike, it seems like switching `ADS_MAX_PARAMS` from `2000` to `10000` in `st.cmd` was enough to fix this. I also multiplied the `ADS_SAMPLE_MS` and `ADS_MAX_DELAY_MS` by the same factor of 5 to compensate (to 250 and 500 respectively). Mike clarified that this would not impact PLC performance much, but it could cause PVs to update at a delayed rate.

Including the KMONO, this IOC currently has `2304` parameters feeding into `5608` PVs, only just over the normal parameter threshold. I do not know why the number of PVs is exactly `1000` more than double the parameter count, I did not expect this. PVs were counted from the `IOC.pvlist` file and params were counted in the IOC logfile where the last param gets index `2304`.

I want to make pytmc dynamically increase this `ADS_MAX_PARAMS` field because it should know how many slots it needs... Maybe this deserves a quick look. The only problem left for this repo for now is that typing `make` resets the changes I made to `st.cmd`.

After this there are still pvs coming up invalid. I plan to investigate some of this tomorrow. Current counts broken down by rough type:
```
Invalid:
- 1298 output records
- 115 input records
Valid:
- 1026 output records
- 3105 input records
- 10 asyn records
- 54 motor records
```
